packages/enrichPrescriptions/tests/testStatusUpdate.test.ts

```diff
@@ -190,4 +196,130 @@ describe("Unit tests for statusUpdate", function () {
 
     expect(requestBundle).toEqual(expectedResponseBundle)
   })
+
+  it("status update does not replace prescriber cancelled status", async () => {
```

Comment:
Does this one need to be a bit more generic in terms of the status? It implies that Prescriber Cancelled is a special case, when it's just any status other than 'With Pharmacy but Tracking not Supported'.